src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/PackageFeeds/MultiSourcePackageFeed.cs

```diff
@@ -19,6 +19,8 @@ namespace NuGet.PackageManagement.VisualStudio
     /// </summary>
     public sealed class MultiSourcePackageFeed : IPackageFeed
     {
+        private const string NUGET_ORG_SOURCE_NAME = "nuget.org";
```

Comment:
We already have this constant defined ( i think as part of protocol project or common). You can use that only.